src/FsAutoComplete.Core/FileSystem.fs

```diff
@@ -412,7 +460,7 @@ type FileSystem(actualFs: IFileSystem, tryFindFile: string<LocalPath> -> Volatil
       |> Utils.normalizePath
       |> tryFindFile
       |> Option.map (fun f -> f.Touched)
-      |> Option.defaultValue DateTime.MinValue
+      |> Option.defaultWith (fun () -> actualFs.GetLastWriteTimeShim f)
```

Comment:
This is the change the _should_ fix typecheck caching issues

Reply:
oh geez, this would have impacted everyone, huh?